app/forms/company/CompanyUniqueTaxReferenceFormProvider.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms.company
+
+import forms.FormErrorHelper
+import forms.mappings.Mappings
+import javax.inject.Inject
+import play.api.data.Form
+
+class CompanyUniqueTaxReferenceFormProvider @Inject() extends FormErrorHelper with Mappings {
+
+  def apply(): Form[String] =
+    Form(
+      "value" -> text("companyUniqueTaxReference.error.required")
```

Comment:
I believe UTRs are only numeric characters (should be in the ticket)

Reply:
Ollie is fixing that in the company-utr branch